tests/integration/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from rh_ui.app_setup import create_app
+from tests.integration.utilities import populate_database, populate_firestore, truncate_database
+
+
+@pytest.fixture
+def app():
+    app = create_app()
+    return app
+
+
+@pytest.fixture
+def test_client(app):
+    test_client = app.test_client()
+    with app.app_context():
+        yield test_client
```

Comment:
I don't think we need to copy these in multiple places, we should be able to keep one `conftest.py` at the top level of the repo (or in `./tests`) for common fixtures